exec/java-exec/src/main/java/org/apache/drill/exec/coord/zk/ZKClusterCoordinator.java

```diff
@@ -252,6 +252,8 @@ public DrillbitEndpoint apply(ServiceInstance<DrillbitEndpoint> input) {
           builder.append(bit.getControlPort());
           builder.append(':');
           builder.append(bit.getDataPort());
+          builder.append(". Version - ");
+          builder.append(bit.getVersion());
```

Comment:
Now that we have many fields, would this be better displayed as a table with columns for each field?

Host | User Port | Control Port | Data Port | Version | ...

In this form we could also show the HTTP port and the port (forgot the name) whose value is computed rather than configured. Eventually, we can also show Drillbit status, etc.